### PR TITLE
Replace old Maestro url with the new one

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/3094

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [x] Internal Infrastructure Improvements

### Release Note Description
Use the new maestro urls in Darc and BarViz

### CODE REVIEW

File: src/Maestro/Maestro.Web/Properties/launchSettings.json [hunk 1]
```diff
@@ -18,12 +10,21 @@
     },
     "Maestro.Web": {
       "commandName": "Project",
+      "commandLineArgs": "get-buil",
```
Comment (premun) [inline src/Maestro/Maestro.Web/Properties/launchSettings.json:13]:
```suggestion
      "commandLineArgs": "get-build",
```
?

Comment (premun) [reply src/Maestro/Maestro.Web/Properties/launchSettings.json:13]: Actually maybe this whole file can get reverted?

Comment (dkurepa) [reply src/Maestro/Maestro.Web/Properties/launchSettings.json]: yup, not this was a typo

**Review by premun — APPROVED** [review]

**Comment by dkurepa** [comment]
won't be merging these changes until the Maestro Prod GH app is updated with the new callback URL https://github.com/dotnet/arcade-services/issues/3039

**Comment by MilenaHristova** [comment]
Rolled out with https://github.com/dotnet/arcade-services/issues/3108

**Pull request merged**